task/transcode.go

```diff
@@ -226,6 +226,23 @@ out:
 	if err != nil {
 		return nil, err
 	}
+	// Download our transcoded output file
+	outputFileInfoReader, err := tctx.outputOS.ReadData(ctx, fullPath)
+	if err != nil {
+		return nil, fmt.Errorf("error reading transcoded file from output OS path=%s err=%w", fullPath, err)
+	}
+	transcodedFile, err := readFile(outputFileInfoReader)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Now with https://github.com/livepeer/task-runner/pull/16/commits/d46c3ec70746c63d7525a838567717816caae23f we don't need to re-download the file here. Can just use `.Reader()` again right?